include/SSVOpenHexagon/Core/HexagonGame.hpp

```diff
@@ -63,6 +63,11 @@ class HexagonGame
     ssvs::GameWindow& window;
 
 public:
+    bool discordHung{false};
+    bool steamHung{false};
+    int8_t discordAttempt{1};
+    int8_t steamAttempt{1};
```

Comment:
```suggestion
    std::int8_t discordAttempt{1};
    std::int8_t steamAttempt{1};
```

Reply:
Also please include the `<cstdint>` header, that's where `std::int8_t` is defined.